src/inventorymanager.h

```diff
@@ -117,15 +117,18 @@ class InventoryManager
 	virtual void inventoryAction(InventoryAction *a){}
 };
 
-#define IACTION_MOVE 0
-#define IACTION_DROP 1
-#define IACTION_CRAFT 2
+enum IAction : u16
+{
+	IACTION_MOVE = 0,
+	IACTION_DROP,
+	IACTION_CRAFT
+};
```

Comment:
* An emun class would be better (although this would require more changes due to the naming change).
* Is the underlying type specification really necessary, or can you let the compiler figure it out?
* The `= 0` is unnecessary, the first item is guaranteed to correspond to 0 by the specification.
* The opening brace should be on the same line as the header.
Example:
```C++
enum class IAction {
        Move,
        Drop,
        Craft,
}
```